e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -22,10 +23,24 @@ public class PublishNodesFromFileTestTheory {
         private readonly ITestOutputHelper _output;
         private readonly IIoTPlatformTestContext _context;
 
+        /// <summary>
+        /// Deployment for edgeHub and edgeAgent so called "base deployment"
+        /// </summary>
+        private readonly IoTHubEdgeBaseDeployment _ioTHubEdgeBaseDeployment;
+
+        /// <summary>
+        /// Deployment for OPC Publisher as standalone
+        /// </summary>
+        private readonly IoTHubPublisherDeployment _ioTHubPublisherDeployment;
+
         public PublishNodesFromFileTestTheory(IIoTPlatformTestContext context, ITestOutputHelper output) {
             _output = output ?? throw new ArgumentNullException(nameof(output));
             _context = context ?? throw new ArgumentNullException(nameof(context));
             _context.OutputHelper = _output;
+
+            // Create deployments.
+            _ioTHubEdgeBaseDeployment = new IoTHubEdgeBaseDeployment(_context);
+            _ioTHubPublisherDeployment = new IoTHubPublisherDeployment(_context);
```

Comment:
I would introduce a second context e.g. `IIoTStandaloneTestContext` because  those classes are required by all standalone theories that will be build in the future, it is not only this class

Reply:
Added `IIoTStandaloneTestContext`, which inherits from `IIoTPlatformTestContext` and adds deployments.